pop_tools/grid.py

```diff
@@ -80,103 +77,145 @@ def get_grid(grid_name, scrip=False):
     dso = xr.Dataset()
-    dso['TLAT'] = xr.DataArray(
-        TLAT / deg2rad,
-        dims=('nlat', 'nlon'),
-        attrs={'units': 'degrees_north', 'long_name': 'T-grid latitude'},
-    )
-
-    dso['TLONG'] = xr.DataArray(
-        TLONG / deg2rad,
-        dims=('nlat', 'nlon'),
-        attrs={'units': 'degrees_east', 'long_name': 'T-grid longitude'},
-    )
-
-    dso['ULAT'] = xr.DataArray(
-        ULAT / deg2rad,
-        dims=('nlat', 'nlon'),
-        attrs={'units': 'degrees_north', 'long_name': 'U-grid latitude'},
-    )
-
-    dso['ULONG'] = xr.DataArray(
-        ULONG / deg2rad,
-        dims=('nlat', 'nlon'),
-        attrs={'units': 'degrees_east', 'long_name': 'U-grid longitude'},
-    )
-
-    dso['DXT'] = xr.DataArray(
-        DXT,
-        dims=('nlat', 'nlon'),
-        attrs={
-            'units': 'cm',
-            'long_name': 'x-spacing centered at T points',
-            'coordinates': 'TLONG TLAT',
-        },
-    )
-
-    dso['DYT'] = xr.DataArray(
-        DYT,
-        dims=('nlat', 'nlon'),
-        attrs={
-            'units': 'cm',
-            'long_name': 'y-spacing centered at T points',
-            'coordinates': 'TLONG TLAT',
-        },
-    )
-
-    dso['TAREA'] = xr.DataArray(
-        TAREA,
-        dims=('nlat', 'nlon'),
-        attrs={'units': 'cm^2', 'long_name': 'area of T cells', 'coordinates': 'TLONG TLAT'},
-    )
-
-    dso['KMT'] = xr.DataArray(
-        KMT,
-        dims=('nlat', 'nlon'),
-        attrs={'long_name': 'k Index of Deepest Grid Cell on T Grid', 'coordinates': 'TLONG TLAT'},
-    )
-
-    dso['z_t'] = xr.DataArray(
-        z_t,
-        dims=('z_t'),
-        name='z_t',
-        attrs={
-            'units': 'cm',
-            'long_name': 'depth from surface to midpoint of layer',
-            'positive': 'down',
-        },
-    )
-
-    dso['dz'] = xr.DataArray(
-        dz,
-        dims=('z_t'),
-        coords={'z_t': dso.z_t},
-        attrs={'units': 'cm', 'long_name': 'thickness of layer k'},
-    )
-
-    dso['z_w'] = xr.DataArray(
-        z_w,
-        dims=('z_w'),
-        attrs={
-            'units': 'cm',
-            'positive': 'down',
-            'long_name': 'depth from surface to top of layer',
-        },
-    )
-
-    dso['z_w_bot'] = xr.DataArray(
-        z_w_bot,
-        dims=('z_w_bot'),
-        attrs={
-            'units': 'cm',
-            'positive': 'down',
-            'long_name': 'depth from surface to bottom of layer',
-        },
-    )
-
-    dso.attrs = grid_attrs
     if scrip:
-        raise NotImplementedError('SCRIP format not implemented')
+        corner_lat, corner_lon = _compute_corners(ULAT, ULONG)
+
+        dso['grid_dims'] = xr.DataArray(np.array([nlon, nlat], dtype=np.int32), dims=('grid_rank',))
+        dso.grid_dims.encoding = {'dtype': np.int32, '_FillValue': None}
+
+        dso['grid_center_lat'] = xr.DataArray(
+            TLAT.reshape((-1,)) / deg2rad, dims=('grid_size'), attrs={'units': 'degrees'}
```

Comment:
Just out of curiosity, would `np.rad2deg()` function be more straightforward here? It took me a few seconds to realize that we are converting from radians to degrees instead of the other way around. 

I don't have any problem with the current solution though